wp-content/themes/citylimits/less/footer.less

```diff
@@ -0,0 +1,164 @@
+.footer-bg {
+    background-color: @white;
+    .footer-skyline {
+        background-image: url("../img/footer-skyline.png");
+        background-color: @black;
+        background-repeat: no-repeat;
+        background-position: bottom;
+        width: 100%;
+        height: 200px;
+        @media (max-width: 769px) {
+            background-position-x: 34%;
+        }
+    }
+    #site-footer {
+        color: @black;
+        #supplementary {
+            padding: 40px 0;
+            .widget-area {
+                .widget_text {
+                    margin-right: -10%;
```

Comment:
This is causing the border of the widget to overlap with the next widget's content:

<img width="620" alt="Screen Shot 2019-10-18 at 16 54 58 " src="https://user-images.githubusercontent.com/1754187/67127846-25b6c900-f1c8-11e9-9ae6-99dec981614a.png">

Reply:
@benlk Do you have your footer layout set to the 3 `span4` column layout?

Reply:
Yes. The problem doesn't change when I switch between Text and Custom HTML widgets. 

<img width="1242" alt="Screen Shot 2019-10-18 at 17 16 33 " src="https://user-images.githubusercontent.com/1754187/67128900-0ff6d300-f1cb-11e9-880d-b90b3e119064.png">

Oh, this is only happening in Firefox. I wonder if it's a box-sizing issue.

Reply:
Nope, it doesn't change with `box-sizing: border-box`.

Reply:
By comparison, here it is without the `-10%` margin-right: 
<img width="1277" alt="Screen Shot 2019-10-18 at 18 06 46 " src="https://user-images.githubusercontent.com/1754187/67131299-1dfc2200-f1d2-11e9-9c3d-485513a3d86f.png">

Reply:
@benlk That was happening because I had the nav div widths set to `width: fit-content;` which FF doesn't like. https://github.com/INN/umbrella-citylimits/pull/66/commits/c2071be203b1c1b64537adb79ef38c0173771132 updates it to `width: max-content;`

Reply:
From about 770px viewport width to at least 900px, the border still impinges upon the menu to the right, and below about 810px the rightmost menu items have the potential to stretch beyond the viewport.

plain screenshot:
![Screen Shot 2019-10-21 at 10 15 14 ](https://user-images.githubusercontent.com/1754187/67213467-10ba7f80-f3ec-11e9-9676-17cd4b37a182.png)

with https://pesticide.io showing element outlines:
![Screen Shot 2019-10-21 at 10 15 31 ](https://user-images.githubusercontent.com/1754187/67213468-10ba7f80-f3ec-11e9-9b5c-1204841b333b.png)

`max-content` was the cause of problems in https://github.com/INN/umbrella-citylimits/issues/43#issuecomment-544314615 as well.

Reply:
@benlk https://github.com/INN/umbrella-citylimits/pull/66/commits/33b6b5687bf97e500285ae5bba02c50a3b574f58 updates the styling so that the first widget area doesn't contain the border or have any of the weird margins and paddings. Instead, it adds a `border-left` to the 2nd `widget-area`

Reply:
Border looks okayish now, but the longer list items are still going over the edge from about 900px down to the breakpoint where it collapses to one column.

I guess we can revisit it when we know what the actual, non-placeholder text looks like on staging/prod.

![Screen Shot 2019-10-21 at 13 22 08 ](https://user-images.githubusercontent.com/1754187/67227863-1ae97780-f406-11e9-9443-b6b73fe944f0.png)